pom.xml

```diff
@@ -16,15 +16,22 @@ See the Apache License Version 2.0 for the specific language governing permissio
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
     <modelVersion>4.0.0</modelVersion>
-
-    <groupId>com.regnosys.rosetta</groupId>
-    <artifactId>com.regnosys.rosetta.code-generators</artifactId>
-    <packaging>jar</packaging>
+    <groupId>com.regnosys.rosetta.code-generators</groupId>
+    <artifactId>parent</artifactId>
```

Comment:
I think the `artefactId` should remain as it was. Did you change it for some technical constraint imposed by Maven?